tests/test_fit_result.py

```diff
@@ -68,27 +79,88 @@ def test_no_constraint(minimizer, fit_kwargs, expected_par, expected_std, expect
          {'objective_value': -float('inf'), 'log_likelihood': float('inf'), 'likelihood': float('inf')})
     ])
 def test_constraints(minimizer, fit_kwargs, expected_par, expected_std, expected_obj, expected_got):
+    """
+    Tests the FitResults from fitting a simple model with Le as constraint
+    using several different minimizers and objectives.
+    """
+
+    model=Model({y: a * x ** b})
+    constraints = [
+        Le(a, b)
+    ]
+    fit = Fit(**fit_kwargs, minimizer=minimizer, model=model, constraints=constraints)
+    fit_result = fit.execute()
+    _run_tests(fit_result, expected_par, expected_std, expected_obj, expected_got)
+
+
+@pytest.mark.parametrize('minimizer',
+    # Added None to check the defaul objective
+    # Removed TrustConstr, because it cannot handle CNM
```

Comment:
s/CNM/CallableNumericalModel/
And add *why*

Reply:
Do you mean to replace `CNM` by `CallableNumericalModel`? :D